src/renderer/components/dock/dock.tsx

```diff
@@ -75,6 +86,22 @@ class NonInjectedDock extends React.Component<Props & Dependencies> {
     this.element?.current.focus();
   };
 
+  nextTab = (direction: Direction = 1) => {
```

Comment:
I don't think this parameter should be defaulted.

Reply:
nit: I also think this function should have a better name, something like `switchToNextTab`

Reply:
Changed